static/js/admin.js

```diff
@@ -163,7 +206,49 @@ exports.setup_page = function () {
         $("#deactivation_stream_modal .stream_name").text(stream_name);
         $("#deactivation_stream_modal").modal("show");
     });
-
```

Comment:
We usually have blank lines between handler blocks here, can you re-add this one (And in a similar place below)?